elastic/integrations
# Added first version of the integration for Elastic Package Registry

## What does this PR do?

Add first version of the integration for Elastic Package Registry. This integration is based on Prometheus Metricbeat module to collect all Prometheus metrics exposed by the Package Registry service. All these metrics are being stored under `metrics-*` indexes.

Tested with Elastic stack 8.0.0 and 8.3.2.

## Checklist



## How to test this PR locally


Testing is based on elastic-package command:

1. Boot up the Elastic stack using [elastic-package](https://github.com/elastic/elastic-package) from `packages/elastic_package_registry` folder.
2. Boot up the EPR service by using the command: `elastic-package service up` inside the `packages/elastic_package_registry` folder
3. Get the container name to be used in configuration. It should be something similar to: `elastic-package-service_elastic_package_registry_1`
4. Add EPR integration in Kibana:
    - Set as hosts: `http://elastic-package-service_elastic_package_registry_1:9100`
    - Go to Discover and check in `metrics-*`. It can be filtered by:
        - `service.type: elastic_package_registry`
        - `data_stream.dataset: elastic_package_registry.metrics`
 

## Related issues

Relates https://github.com/elastic/package-registry/issues/797

## Screenshots

Screenshots for kibana 8.0.0:


![elastic_package_registry-metrics-overview](https://user-images.githubusercontent.com/5330827/180475516-5fbcc2f4-003e-4bb3-bdf5-c74e73c652fe.png)


![EPR tile details](https://user-images.githubusercontent.com/5330827/180475465-17d880ee-756d-4a8b-aa32-87e190580185.png)
